src/actions/account.js

```diff
@@ -545,6 +547,7 @@ export const switchAccount = (accountId) => async (dispatch, getState) => {
     dispatch(handleRefreshUrl())
     dispatch(staking.clearState())
     dispatch(refreshAccount())
+    dispatch(tokens.clearState())
```

Comment:
wonder whether we want to have `clearState` explicit vs handling this in reducer for `switchAccount`, etc

feel like if only goal is to update state, then it makes sense to update state in whatever actions reducer vs spin off more actions to do this

unless I'm missing smth important

Reply:
You're right, but I would leave it for now, because I would like to change it completely while refactoring the redux state - then the account state will not be cleared at all